test/test_optim.py

```diff
@@ -578,12 +578,15 @@ def test_multi_tensor_optimizers(self):
         self._test_derived_optimizers(optimizer_pairs_with_flags, "foreach")
 
     def test_fused_optimizers(self):
-        optimizer_pairs_with_flags = [
```

Comment:
Do we have a test that ensures that we raise a nice error if you pass `foreach=True, fused=True` ?

Reply:
no, at the moment.